test/rubocop-diff-test

```diff
@@ -48,44 +42,44 @@ cp "$current_path"/Gemfile.lock .
 
 mkdir -p vendor
 
-if  [[ -n "${GITHUB_WORKSPACE}" ]];then
+if  [ -n "${GITHUB_WORKSPACE}" ];then
   ln -s "${GITHUB_WORKSPACE}/"vendor/bundle vendor/bundle
```

Comment:
I'd put the whole path in interpolating quotes: `ln -s "${GITHUB_WORKSPACE}/vendor/bundle vendor/bundle"`, likewise in several places below